[Rust] Bug: Body parameter name ignored

##### Description

Body parameters name is not used when serializing the body. Generated body parameter name is always assumed to be `body`. See [here.](https://github.com/swagger-api/swagger-codegen/blob/5e9d977a03756b0445f026dec0a4f1b6095ff8b1/modules/swagger-codegen/src/main/resources/rust/api.mustache#L71)

##### Swagger-codegen version

2.3.0

##### Swagger declaration file content or url

```yaml
paths:
  '/ExampleResource':
    post:
      summary: Example
      parameters:
        - in: body
          name: body_param_name
          schema:
            $ref: '#/definitions/Example'
      responses:
        '200':
          description: Successful operation.
definitions:
  Example:
    description: "example"
    type: string
```

##### Command line used for generation

`./run-in-docker.sh generate -i example.yaml -l rust -o /gen/out/rust-example -DpackageName=example`

##### Steps to reproduce

1. Generate the code
2. Attempt to compile, receive error:

```
error[E0425]: cannot find value `body` in this scope
  --> src/apis/default_api.rs:54:49
   |
54 |         let serialized = serde_json::to_string(&body).unwrap();
   |                                                 ^^^^ not found in this scope
```

##### Suggest a fix/enhancement

Use `{{paramName}}` instead of `body`.

--- a/modules/swagger-codegen/src/main/resources/rust/api.mustache
+++ b/modules/swagger-codegen/src/main/resources/rust/api.mustache
@@ -75,7 +75,7 @@ impl<C: hyper::client::Connect>{{classname}} for {{classname}}Client<C> {
 
         {{#hasBodyParam}}
         {{#bodyParams}}
-        let serialized = serde_json::to_string(&body).unwrap();
+        let serialized = serde_json::to_string(&{{paramName}}).unwrap();
         req.headers_mut().set(hyper::header::ContentType::json());
         req.headers_mut().set(hyper::header::ContentLength(serialized.len() as u64));
         req.set_body(serialized);

